### PR TITLE
use drop shadow filter

### PR DESCRIPTION
**Pull request by Haroenv**
this will work for the edges of the text as wel as for the edges of the svg and thus avoids showing no interaction with the image while still avoiding loading both logo and logo-hover.

However I've noticed that it's not easy to define two shadows with this filter

TODO:
- [x] usage of drop shadow filter
- [ ] testing compatibility
- [ ] making a "double" shadow work

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -47,7 +47,8 @@ h1 {
   color: #000000;
   display: inline-block;
   position: relative;
-  text-shadow: 1px 1px 0 #EDEDED, 3px 3px 0 rgba(0, 0, 0, 0.05)
+  filter: drop-shadow(3px 3px 0 rgba(0, 0, 0, 0.05));
```
Comment (LukyVj) [inline css/style.css:52]: Shouldn't you use prefixed css ?

File: css/style.css [hunk 2]
```diff
@@ -47,7 +47,8 @@ h1 {
...
+  transition: .2s filter ease-in-out;
```
Comment (LukyVj) [inline css/style.css:54]: 0.2s

File: index.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
   <header>
     <h1>
       <a href="https://github.com/Haroenv/holmes/">Holmes</a>
-      <img src="images/holmes_logo-hover.svg" alt="Holmes Logo">
+      <img src="images/holmes_logo.svg" alt="Holmes Logo">
```
Comment (LukyVj) [inline index.html:17]: Good one <3

File: css/style.css (hunk 1 above)
Comment (Haroenv) [inline css/style.css:52]: that's an outdated diff

**Comment by LukyVj** [comment]
Y U NO MERGE ?

**Pull request closed without merging**